### PR TITLE
Toggle

### LINKED ISSUE #8: Toggle function
## Value Proposition

As a User
I would like to use the arrow to toggle the work report between large and small 
​

## Description

​

Image

## Acceptance Criteria

- [ ] Toggle function to hide information on page 2.

## Tasks

- [x] new branch
- [x] add component / button to hide information
- [x] write test

### PR DESCRIPTION
**Pull request by Kachev**

### CODE REVIEW

File: components/ReportsLists/index.js [hunk 1]
```diff
@@ -27,40 +29,92 @@ const StyledLi = styled.li`
...
+      ? "overflow: hidden;text-overflow: ellipsis;display: -webkit-box;-webkit-line-clamp: 7; -webkit-box-orient: vertical;"
+      : ""};
 `;
 const StyledUl = styled.ul`
   display: flex;
   flex-direction: column;
 `;
 const StyledSection = styled.section`
-  display: flex;
+  /*   display: flex;
```
Comment (XxKikoxX) [inline components/ReportsLists/index.js]: you should delete the comments if you don't use this feature

File: components/ReportsLists/index.js [hunk 2]
```diff
@@ -27,40 +29,92 @@ const StyledLi = styled.li`
...
   border: 2px solid black;
   justify-content: space-around;
-  flex-wrap: wrap;
+  flex-wrap: wrap; */
 `;
 const StyledContainer = styled.div`
-  display: flex;
+  /*  display: flex;
```
Comment (XxKikoxX) [inline components/ReportsLists/index.js]: same here with the comments

File: components/ReportsLists/index.js [hunk 3]
```diff
@@ -27,40 +29,92 @@ const StyledLi = styled.li`
...
   flex-direction: column;
   border: 2px solid black;
-  padding: 5px;
+  padding: 5px; */
```
Comment (XxKikoxX) [inline components/ReportsLists/index.js]: and here

**Review by XxKikoxX — COMMENTED** [review]
the app looks solid to me
I also think that this app could actually solve the paperwork problem, which can be impractical for craftsmen in everyday life and it's not healthy for our planet either, so keep it up

**Review by toshydev — APPROVED** [review]
Very good work! LGTM! ✅

**Pull request merged**